scripts/presubmit-tests.sh

```diff
@@ -169,16 +169,22 @@ function default_build_test_runner() {
     # Consider an error message everything that's not a package name.
     errors_go1="$(grep -v '^\(github\.com\|knative\.dev\)/' "${report}" | sort | uniq)"
   fi
-  # Get all build tags in go code (ignore /vendor)
+  # Get all build tags in go test code (ignore /vendor)
```

Comment:
`tags` is not restricted to content in `/test`.

```suggestion
  # Get all build tags in go code (ignore /vendor)
```